pallets/msa/src/rpc/src/lib.rs

```diff
@@ -36,17 +36,17 @@ pub trait MsaApi<BlockHash, AccountId> {
 	#[method(name = "msa_checkDelegations")]
 	fn check_delegations(
 		&self,
-		delegator_msa_ids: Vec<MessageSourceId>,
-		provider_msa_id: MessageSourceId,
+		delegator_msa_ids: Vec<DelegatorId>,
+		provider_msa_id: ProviderId,
 		block_number: Option<BlockNumber>,
-	) -> RpcResult<Vec<(MessageSourceId, bool)>>;
+	) -> RpcResult<Vec<(DelegatorId, bool)>>;
 
 	/// Retrieve the list of currently granted schemas given a delegator and provider pair
 	#[method(name = "msa_grantedSchemaIdsByMsaId")]
 	fn get_granted_schemas_by_msa_id(
 		&self,
-		delegator_msa_id: MessageSourceId,
-		provider_msa_id: MessageSourceId,
+		delegator_msa_id: DelegatorId,
+		provider_msa_id: ProviderId,
```

Comment:
Custom RPCs are not automatic, so this would require an update to https://github.com/LibertyDSNP/frequency/blob/main/js/api-augment/definitions/msa.ts

Reply:
Thought I caught these, thanks.